src/styles/variables.scss

```diff
@@ -0,0 +1,15 @@
+$calendar-day-bg-color: #eee;
+$calendar-day-border-color: #000;
+$calendar-day-hover-color: pink;
+$day-heigth: 100px;
+$day-width: 100px;
+$calendar-width: 7 * $day-width + 10px;
+$month-length: 28,29,30,31;
+$main-transition: all 0.5s;
```

Comment:
It is a good practice to specify the properties you want apply the transition to instead of `all`.
In that case, I think it's better to create a variable just for `transition-duration`
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x